keyboards/akb/raine/info.json

```
@@ -0,0 +1,84 @@
+                { "label": "Insert", "x": 15.25, "y": 0 },
+                { "label": "Tab", "x": 0, "y": 1, "w": 1.5 },
+                { "label": "Q", "x": 1.5, "y": 1 },
+                { "label": "W", "x": 2.5, "y": 1 },
+                { "label": "E", "x": 3.5, "y": 1 },
+                { "label": "R", "x": 4.5, "y": 1 },
+                { "label": "T", "x": 5.5, "y": 1 },
+                { "label": "Y", "x": 6.5, "y": 1 },
+                { "label": "U", "x": 7.5, "y": 1 },
+                { "label": "I", "x": 8.5, "y": 1 },
+                { "label": "O", "x": 9.5, "y": 1 },
+                { "label": "P", "x": 10.5, "y": 1 },
+                { "label": "|", "x": 11.5, "y": 1, "w": 1.5 },
+                { "label": "7", "x": 13.25, "y": 1 },
+                { "label": "8", "x": 14.25, "y": 1 },
+                { "label": "9", "x": 15.25, "y": 1 },
+                { "label": "Caps", "x": 0, "y": 2, "w": 1.75 },
+                { "label": "A", "x": 1.75, "y": 2 },
+                { "label": "S", "x": 2.75, "y": 2 },
+                { "label": "D", "x": 3.75, "y": 2 },
+                { "label": "F", "x": 4.75, "y": 2 },
+                { "label": "G", "x": 5.75, "y": 2 },
+                { "label": "H", "x": 6.75, "y": 2 },
+                { "label": "J", "x": 7.75, "y": 2 },
+                { "label": "K", "x": 8.75, "y": 2 },
+                { "label": "L", "x": 9.75, "y": 2 },
+                { "label": "~", "x": 10.75, "y": 2 },
+                { "label": "ENTER", "x": 11.75, "y": 2, "w": 1.25 },
+                { "label": "4", "x": 13.25, "y": 2 },
+                { "label": "5", "x": 14.25, "y": 2 },
+                { "label": "6", "x": 15.25, "y": 2 },
+                { "label": "Shift", "x": 0, "y": 3, "w": 1.25 },
+                { "label": "|", "x": 1.25, "y": 3 },
+                { "label": "Z", "x": 2.25, "y": 3 },
+                { "label": "X", "x": 3.25, "y": 3 },
+                { "label": "C", "x": 4.25, "y": 3 },
+                { "label": "V", "x": 5.25, "y": 3 },
+                { "label": "B", "x": 6.25, "y": 3 },
+                { "label": "N", "x": 7.25, "y": 3 },
+                { "label": "M", "x": 8.25, "y": 3 },
+                { "label": "?", "x": 9.25, "y": 3 },
+                { "label": "Shift", "x": 10.25, "y": 3, "w": 1.5 },
+                { "label": "1", "x": 13.25, "y": 3 },
+                { "label": "2", "x": 14.25, "y": 3 },
+                { "label": "3", "x": 15.25, "y": 3 },
+                { "x": 12, "y": 3.25 },
+                { "label": "Ctrl", "x": 0, "y": 4, "w": 1.25 },
+                { "label": "Alt", "x": 2.25, "y": 4, "w": 1.25 },
+                { "label": "SPLEFT", "x": 3.5, "y": 4, "w": 2.25 },
+                { "label": "7U", "x": 5.75, "y": 4 },
+                { "label": "SPRIGHT", "x": 6.75, "y": 4, "w": 1.75 },
+                { "label": "Menu", "x": 9.5, "y": 4, "w": 1.25 },
+                { "label": "0", "x": 14.25, "y": 4 },
```

Whoops, forgot to hit submit on this bit.

```suggestion
                { "x": 11, "y": 4.25 },
                { "x": 12, "y": 4.25 },
                { "x": 13, "y": 4.25 },
                { "label": "0", "x": 14.25, "y": 4 },
```